plugins/api/src/main/kotlin/PluginConfig.kt

```diff
@@ -40,6 +40,13 @@ data class PluginConfig(
     @JsonIgnore
     val secrets: Options = emptyMap()
 ) {
+    companion object {
+        /**
+         * Constant for an empty [PluginConfig] that contains no options and no secrets. This constant can be used as a
```

Comment:
Is it worth saying "that contains no options and no secrets"? That would need to get adapter if we ever add more properties to the class.